### DISCUSSION TITLE
Does the preview feature not support nested folders?

### DISCUSSION THREAD

**Discussion by affanshahid** [resend/react-email#1109] [Q&A]
The preview feature (`email dev`) does not pick up emails in nested folders, i.e `emails/foo/bar.tsx` instead of just `emails/bar.tsx`

**Comment by TomQuast** [comment]
I would like to know that too. I am using react-mail for creating mails for several projects and right now I have to prefix the filenames. It would be nice to be able to organize them into folders instead :)
